src/Illuminate/Database/Eloquent/Relations/BelongsToMany.php

```diff
@@ -784,7 +784,7 @@ public function findOr($id, $columns = ['*'], ?Closure $callback = null)
      * @param  mixed  $operator
      * @param  mixed  $value
      * @param  string  $boolean
-     * @return \Illuminate\Database\Eloquent\Model|static
+     * @return \Illuminate\Database\Eloquent\Model|static|null
```

Comment:
```suggestion
     * @return ?\Illuminate\Database\Eloquent\Model
```

Unless I'm mistaken, this should be correct